scripts/bsd/netbsd-prep-postgres.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+
+# Install required packages for running tests on netBSD
+PKG_PATH="http://cdn.NetBSD.org/pub/pkgsrc/packages/NetBSD/$(uname -p)/$(uname -r|cut -f '1 2' -d.)/All/" && \
+export PKG_PATH && \
+/usr/sbin/pkg_add pkgin && \
+/usr/pkg/bin/pkgin update && \
+mkdir /usr/pkg/pgsql && \
+/usr/pkg/bin/pkgin -y install \
+curl \
+vim \
+git \
+mozilla-rootcerts \
+gmake \
+gettext \
+meson \
+bison \
+ccache \
+p5-IPC-Run \
+flex \
+pkgconf \
+python38 \
+icu \
+lz4 \
+libxslt \
+tcl && \
```

Comment:
I think it might be a good idea to have two different "levels" of images. One that just defines a basic working open/netbsd GCP image, and then a second one installing all the postgres dependencies, ontop of the prior image. That way other projects can use the base image (including hopefully getting the cirrus folks to automatically test their support for open / netbsd).